components/scripts/lib/build-scan-offline.sh

```diff
@@ -34,9 +34,20 @@ find_build_scan_dump() {
 }
 
 read_build_scan_dumps() {
-  local build_scan_csv
-  echo -n "Extracting build scan data"
-  build_scan_csv="$(invoke_java "$BUILD_SCAN_SUPPORT_TOOL_JAR" extract "0,${build_scan_dumps[0]}"  "1,${build_scan_dumps[1]}")"
+  local build_scan_csv args
+  args=()
+
+  args+=(
+      "extract"
+      "--license-file" "${SCRIPT_DIR}/gradle-enterprise.license"
+      "1,${build_scan_dumps[1]}"
+      "0,${build_scan_dumps[0]}"
```

Comment:
Do we use the same approach for passing args with the jar file that fetches build scans remotely?

Reply:
Yes, although for the online fetch, this is generated more dynamically. See https://github.com/gradle/gradle-enterprise-build-validation-scripts/blob/422822eec0c2919423a44a5d36eff1c175b0daee/components/scripts/lib/build-scan-online.sh#L100-L102

Reply:
Could we do the same here as we do in the online case to make the code more similar?

Reply:
If this always involves fetching two build scans, I suggest we keep the current code here and adjust the code for the online mode also to pass on two explicit build scans rather than _n_ build scans.

Reply:
I now see that in online mode, we invoke either one or two builds and thus need the generic looping.

I'm withdrawing my suggestion.